### PR TITLE
add key results for global knowledge-sharing

### PR DESCRIPTION
**Pull request by quiddity-wp**

### CODE REVIEW

File: src/terms.js [hunk 1]
```diff
@@ -132,11 +132,13 @@ export default [
     'flexibility',
     'foot view',
     'four pillars',
+    'frameworks', 
     'front lines',
     'functional',
     'full benefit',
     'game plan',
     'behavioral',
+    'global(ly)',
```
Comment (mourner) [inline src/terms.js]:
```suggestion
    'global(ly)?',
```

File: src/terms.js [hunk 2]
```diff
@@ -168,7 +170,9 @@ export default [
     'intellectual property',
     'intuitive',
     'iot',
+    'key results?', 
     'knowledge (base|transfer)',
+    'knowledge-sharing',
```
Comment (mourner) [inline src/terms.js]: I would merge this with the term above, adding a `.` instead of ` `.

File: src/terms.js [hunk 3]
```diff
@@ -132,11 +132,13 @@ export default [
     'flexibility',
     'foot view',
     'four pillars',
+    'frameworks',
```
Comment (mourner) [inline src/terms.js]: Not sure we should include this one because it can be used in a lot of non-bullshit contexts.

**Comment by quiddity-wp** [comment]
I think that solves it? I'm not sure what the Travis CI error means. Edit freely!

**Pull request merged**